tests/regression/test_garbage.py

```diff
@@ -0,0 +1,12 @@
+from firedrake import *
+from firedrake.petsc import garbage_cleanup
```

Comment:
Is this import needed? `garbage_cleanup` is added to `__all__` in `firedrake.petsc`.

Reply:
No, I think I was in two minds whether it should actually be added to the list of things imported by `from firedrake import *` or whether I want to force users to import it as a "I know what I'm doing and really want to manually clean things up step".

Reply:
What do you think?

Reply:
I like the idea of requiring an explicit import. As you say it implies that it isn't a thing that most users need to do.